utils.php

```diff
@@ -149,3 +150,22 @@ function normalize_term($term)
     }
     return ($year - 1) . "9";
 }
+
+/**
+ * Function checks if user_id and utorid match. If not, returns 401 error
+ *
+ * @throws 401 Unauthorized Access
+ */
+function verify_user_id($parameters)
+{
+    if ($parameters['auth'] != null) {
+        if (
+            ($parameters['user_id'] != $parameters['auth']['utorid']) ||
+            $parameters['user_id'] == null
+        ) {
```

Comment:
Combine with above. 
```
if ($parameters['auth']['utorid'] != null && $paramters['auth']['utorid'] != $parameters['user_id']) {
...throw error...
}
```

And use `$params` instead of `$parameters`. It's easier to read and type.